src/l500/l500-device.cpp

```diff
@@ -615,17 +618,7 @@ namespace librealsense
         if (command_str == "GET-NEST")
         {
             // Handle extended temperature command
-            auto nest_response = _hw_monitor->send(command{ ivcam2::TEMPERATURES_GET });
-            if (nest_response.size() < sizeof(extended_temperatures))
-            {
-                throw invalid_value_exception(to_string() <<
-                    "Extended temperatures get FW command failed, size expected: " << sizeof(extended_temperatures )
-                    << " , size received: " << nest_response.size() );
-            }
-
-            auto const & ext_temp
-                = *(reinterpret_cast<extended_temperatures *>(nest_response.data()));
-            LOG_INFO("Nest AVG: " << ext_temp.nest_avg);
+            LOG_INFO("Nest AVG: " << get_temperatures().nest_avg);
```

Comment:
Shouldn't this be guarded by a FW version check?

Reply:
It should, the viewer check it. 
We can add it inside also..

Reply:
Done